src/main/java/org/jitsi/jicofo/conference/JitsiMeetConferenceImpl.java

```diff
@@ -1511,25 +1512,17 @@ else if (!participant.hasModeratorRights()
             }
         }
 
-        if (doMute
-            && participant.getChatMember().isJigasi()
-            && !participant.hasAudioMuteSupport())
-        {
-            logger.warn("Mute not allowed, toBeMuted is jigasi.");
-            return MuteResult.NOT_ALLOWED;
-        }
-
-
-        if (doMute && participant.getChatMember().isJibri())
+        if (participant.shouldSuppressForceMute())
         {
-            logger.warn("Mute not allowed, toBeMuted is jibri.");
+            logger.warn("Force mute suppressed, returning NOT_ALLOWED:" + participant);
```

Comment:
This case used to only trigger if doMute was true - this will now return NOT_ALLOWED for unmuting a Jigasi too.  Is this what we want?

Reply:
I think the result is equivalent because "unmute" is only allowed when the request comes from the endpoint itself, and since jigasi doesn't support muting it never tries to unmute itself either (besides, it can't be muted) :)

But this was not intenional and I see a little bit of cleanup I can do. If you think the check makes more sense I'll add it back.